Formula/arangodb.rb

```diff
@@ -27,6 +27,8 @@ class Arangodb < Formula
       revision: "e32307e9ae5a0046214cb066355a8577e6fc4148"
   end
 
+  patch :DATA
```

Comment:
We can pick up the commit directly from the tree like this:

```suggestion
  patch do
    url "https://github.com/arangodb/arangodb/commit/e66bf335.patch?full_index=1"
    sha256 "whatever-value"
  end
```

Reply:
Thanks for the hint. Done.